### ISSUE TITLE
Increased urine homocysteine

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11235]
**Preferred term label:**
Increased urine homocysteine

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
MTHFR Deficiency (https://www.ncbi.nlm.nih.gov/books/NBK615089/)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
there is [Homocystinuria HP:0002156](https://hpo.jax.org/browse/term/HP:0002156) -- oxidized dimer form, homocysteine would be the reduced monomer form. I thought that these forms exist in equilibrium and we actually measure total homocyst(e)ine?

**Comment by MickeySegal** [comment]
https://labpedia.net/homocysteinemia-indications-and-interpretations/ says
"Homocysteine does not accumulate in the blood because it is unstable in an aqueous solution; when it is in excess, it will undergo oxidation and give rise to homocystine".  

Since we only have a finding for high levels we can used the merged [Homocystinuria HP:0002156](https://hpo.jax.org/browse/term/HP:0002156).

**Comment by pnrobinson** [comment]
@MickeySegal , OK, closing!
